src/display/annotation_layer.js

```diff
@@ -293,8 +297,10 @@ class LinkAnnotationElement extends AnnotationElement {
     const isRenderable = !!(
       parameters.data.url ||
       parameters.data.dest ||
-      parameters.data.action
+      parameters.data.action ||
+      parameters.data.isTooltipOnly
```

Comment:
Does this actually work as expected? `isTooltipOnly` is only available in button widget annotations, but this is the link annotation class, which is not a button widget annotation?

Reply:
PushButton isn't a widget but a Link so it's why...
I slightly modify the code to put the isTooltipOnly in PushButton